### DISCUSSION TITLE
Duda sobre TP2 - Fibonacci

### DISCUSSION THREAD

**Discussion by alengeor** [INGCOM-UNRN-P1/cursada-2023#64] [Preguntas y respuestas] [answered]
Buenas buenass, como andan?

Leí este ejercicio:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/ffbdd9a0-ae81-49dd-a4d9-6c8c50c3851c)
E inicialmente pensé en hacer una función que retorne el resultado de la _suma de los terminos de la sucesión de fibonacci, hasta el limite n._

Por ejemplo, limite en 5. Sumaría:
2 + 3 + 5 + 8 + 13 = 31.

Mi funcion fibonacci(5) retornaría 31. ¿Esto debería así?

Porque luego, entendí que se deben sumar _los dígitos de cada termino, hasta el termino n_. Entonces quedaría:
2 + 3 + 5 + 8 + (1 + 3) = 22.

Mi funcion fibonacci(5) ahora retornaría 22. ¿Cual opcion es la correcta? _o si ninguna lo es, como sería(?_

Gracias totalesss, buen Viernes!

**Comment by alengeor** [comment]
Perdón, me parece que me confundí con los términos en la sucesión. Dejo un ejemplo de como me quedo el contador de dígitos para fibonacci(7):
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/5f5b9cce-3908-4c67-b0d6-60b6c81aeb80)

Ahorasi, ¿Cumple con la consigna?

**Comment by martinvilu** [comment, marked as answer]
Usando la lista de https://oeis.org/A000045 como base:

Los resultados esperados

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/90c00656-951e-4b0b-978b-896abca59112)



_Si el término 1 es cero, o 1, el ejemplo calcula la suma del término 9, en lugar del 10._

**Reply by martinvilu** [reply]
Cuando termino de Fibonacci tiene un solo dígito, el valor individual coincide, pero la suma debe de incluir los valores anteriores. (la cuarta fila)

**Reply by alengeor** [reply]
A ver ahora:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/faa375e7-96e3-4b10-883f-00caace4d78b)

**Reply by alengeor** [reply]
Otro ejemplo para que llegue a sumar de a dos digitos:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/1f21cea4-e06a-409f-86c1-666326b98bfb)

**Reply by martinvilu** [reply]
Y, compara con la tabla que te pasé
